### ISSUE TITLE
Implement process filtering

### ISSUE THREAD

**Issue by delucca** [cjbassi/ytop#47]
Hi. I'm trying to filter process with the `/` key (like is shown on the `?` window, but when I type `/` nothing happens.

Required information:

- ytop version (`ytop -V`): 0.5.1
- The output of `uname -a`: Linux SPF1LT-PJ000443.ldap.quintoandar.com.br 4.19.106-1-MANJARO #1 SMP Mon Feb 24 08:17:18 UTC 2020 x86_64 GNU/Linux

Include any of the following information if relevant:

- Terminal emulator (e.g. iTerm or gnome terminal): Deepin Terminal
- tmux version (`tmux -V`): 3.0a
- Any relevenat hardware info:

**Comment by cjbassi** [comment]
Yeah the process filtering isn't implemented yet. It's one of the remaining todos listed in the readme. I forgot that I added it to the help menu, I can see how that can be a little confusing.

**Comment by delucca** [comment]
Ah, ok! Thanks for the answer :)

Looking forward to an implementation

**Comment by kawpii** [comment]
Process filtering is a very useful feature

**Comment by gennaro-tedesco** [comment]
Do I understand correctly that the filtering hasn't been implemented yet, then (because I have also been trying to use it as per documentation with no luck)?
